src/app-layout/visual-refresh/tools.scss

```diff
@@ -170,4 +170,6 @@ handleSplitPanelMaxWidth function in the context.
       z-index: 1;
     }
   }
+
+  @include styles.styles-reset;
```

Comment:
The order of `@include` matters, this way it will cancel all declarations above.

it is recommended to always write in this order, to not have any accidental overrides

```css
.show-tools {
  @include styles.styles-reset; 
  box-sizing: border-box;
  padding: awsui.$space-scaled-s awsui.$space-layout-toggle-padding;
}
```

Reply:
This come from SplitPanelStyles, which was the last one of the classname list https://github.com/cloudscape-design/components/pull/839/files#diff-21e6d2c3699425bb1b04025db0219b97b57bf68f7df16d23eccd4737791ee397L124. So I guess there was such overrides before? I am thinking running screenshot tests to see if this style reset is needed at all.

Reply:
Order of classes in HTML does not matter, only order in CSS files matters. If you do not believe - try by yourself